docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/OrdinaryLeastSquaresWithOptions.tt

```diff
@@ -0,0 +1,27 @@
+﻿<#@ include file="RegressionSamplesTemplate.ttinclude"#>
+
+<#+
+string ClassHeader = null;
+string ClassName="OlsWithOptions";
+string Trainer = "Ols";
+string TrainerOptions = @"OlsTrainer.Options
+            {
+                LabelColumnName = nameof(DataPoint.Label),
+                FeatureColumnName = nameof(DataPoint.Features),
+                L2Regularization = 0.1f,
+                CalculateStatistics = false
```

Comment:
your one-line comments for the options in the other files were so nice :)

Reply:
Thanks. But I add them only if I don't need to dig into their code to understand their meanings.

---
In reply to: [275458769](https://github.com/dotnet/machinelearning/pull/3319#discussion_r275458769) [](ancestors = 275458769)